### PR TITLE
Added pt_BR (Brazilian Portuguese) translation

### PR DESCRIPTION
**Pull request by manipuladordedados**

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -21,3 +21,4 @@ Matteo Pasotti <matteo@mageia.it>
 @kulti <kultihell@gmail.com>
 @pisculichi <pisculichi@openmailbox.org>
 Sérgio Marques <smarquespt@gmail.com>
+Valter Nazianzeno <manipuladordedados@gmail.com>
```
Comment (paulolieuthier) [inline AUTHORS]: I'd say one has to contribute code in order to make it to the AUTHORS list, but the two previous guys have also contributed only translations. @agaida, @jleclanche what's the policy here?

Comment (jleclanche) [inline AUTHORS]: The AUTHORS file should have a translators section.

Comment (paulolieuthier) [inline AUTHORS]: @manipuladordedados could you please update that?

Comment (manipuladordedados) [inline AUTHORS]: Claro =)
Ok!

Comment (paulolieuthier) [inline AUTHORS]: Obrigado! ;)

**Pull request merged**